.github/workflows/config.yml

```diff
@@ -368,6 +368,7 @@ jobs:
           PGUSER: postgres
           ETHEREUM_JSONRPC_CASE: "EthereumJSONRPC.Case.Parity.Mox"
           ETHEREUM_JSONRPC_WEB_SOCKET_CASE: "EthereumJSONRPC.WebSocket.Case.Mox"
+          SUBNETWORK: Celo
```

Comment:
Why the variable is SUBNETWORK and not NETWORK?

Reply:
So here `SUBNETWORK` refers to the specific celo env, that is Alfajores Baklava or Celo (mainnet) - whereas `NETWORK` refers to the chain itself, e.g. Ethereum or Celo. Confusingly `Celo` means both "A celo blockchain" and "celo mainnet".

Reply:
Actually this shouldn't be needed anymore, thanks for pointing it out